frontend/src/individual_form.tsx

```diff
@@ -408,6 +404,32 @@ export function IndividualForm({
                     onUpdateIndividual("sex", newValue?.value ?? "");
                   }}
                 />
+                </div>
+              <div className="flexRow">
+              <TextField
+                  required
+                  error={litterError}
+                  label="Antal födda i kullen"
+                  className="control controlWidth"
+                  variant={inputVariant}
+                  value={individual.litter_size ?? ''}
+                  type="number"
+                  onChange={(event) => {
+                    onUpdateIndividual("litter_size", +event.currentTarget.value);
+                  }}
+                />
+                <TextField
+                  required
+                  error={litterError6w}
+                  label="Antal levande i kullen (efter 6 veckor)"
```

Comment:
This label is too long for the form when it is used for creating or updating certificates. Maybe you could shorten it to "Antal levande i kullen efter 6v"?
An alternative would be to increase the width of the popup in which the individual_cert component is rendered, but that might be a hassle...

Reply:
fixee